### PR TITLE
feat: create an azure-ml pipeline for eval_prompts()

### LINKED ISSUE #35: Create an azure-ml pipeline for eval_prompts()
Create an Azure ML flow for eval_prompts() by creating a workflow similar to azureml/eval.yaml

### PR DESCRIPTION
**Pull request by anujsinha3**
## Change Description
Created an azure-ml pipeline for eval_prompts()
closes #35 

- [x] My PR includes a link to the issue that I am addressing

## Code Quality
- [x] I have read the Contribution Guide
- [x] My code follows the code style of this project
- [x] My code builds (or compiles) cleanly without any errors or warnings
- [x] My code contains relevant comments and necessary documentation

### Bug Fix Checklist
- [ ] My fix includes a new test that breaks as a result of the bug (if possible)
- [ ] My change includes a breaking change
  - [ ] My change includes backwards compatibility and deprecation warnings (if possible)

### New Feature Checklist
- [ ] I have added or updated the docstrings associated with my feature using the [NumPy docstring format](https://numpydoc.readthedocs.io/en/latest/format.html)
- [ ] I have updated the tutorial to highlight my new feature (if appropriate)
- [ ] I have added unit/End-to-End (E2E) test cases to cover my new feature
- [ ] My change includes a breaking change
  - [ ] My change includes backwards compatibility and deprecation warnings (if possible)

### Documentation Change Checklist
- [ ] Any updated docstrings use the [NumPy docstring format](https://numpydoc.readthedocs.io/en/latest/format.html)

### Build/CI Change Checklist
- [ ] If required or optional dependencies have changed (including version numbers), I have updated the README to reflect this
- [ ] If this is a new CI setup, I have added the associated badge to the README

### Other Change Checklist
- [ ] Any new or updated docstrings use the [NumPy docstring format](https://numpydoc.readthedocs.io/en/latest/format.html).
- [ ] I have updated the tutorial to highlight my new feature (if appropriate)
- [ ] I have added unit/End-to-End (E2E) test cases to cover any changes
- [ ] My change includes a breaking change
  - [ ] My change includes backwards compatibility and deprecation warnings (if possible)

### CODE REVIEW

File: azureml/eval_prompts.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandJob.schema.json
+command: >
+  python -m autora.doc.pipelines.main eval-prompts
+  ${{inputs.data_dir}}/data.jsonl
+  ${{inputs.data_dir}}/all_prompt.json
```
Comment (carlosgjs) [inline azureml/eval_prompts.yml:5]: Let's make the prompts file its own input parameter. This will make it easier to run experiments.

**Review by carlosgjs — APPROVED** [review]
Just one comment inline.

Also, can you add a link to a run of this workflow in the description of the PR?

File: azureml/eval_prompts.yml (hunk 1 above)
Comment (anujsinha3) [reply azureml/eval_prompts.yml:5]: should I parameterize the data file as well? The default value can be 'data.jsonl' but a user can modify input if they want. WDYT?

**Comment by anujsinha3** [comment]
Example run with test LLM model: https://ml.azure.com/experiments/id/e1882f37-87e8-4ff9-930a-7aa050693072/runs/great_corn_qbslxrsxyn?wsid=/subscriptions/e4f4cccd-ac89-41b8-aae6-896e5a8471e4/resourcegroups/AutoRADoc/workspaces/AutoRAML&tid=f6b6dd5b-f02f-441a-99a0-162ac5060bd2

**Pull request merged**